resources/howto/flst_search/select_options_list.txt
FS_GKNR HTML set

<select id='FS_GKNR' class=''>
<option value=''>Bitte auswählen...</option>
<option value='2110' title='Meißner, LK: Werra-Meißner'>Abterode (2110)</option>
<option value='51' title='Poppenhausen (Wasserkuppe), LK: Fulda'>Abtsroda (51)</option>
<option value='2440' title='Breidenbach, LK: Marburg-Biedenkopf'>Achenbach (2440)</option>
<option value='1855' title='Melsungen, LK: Schwalm-Eder'>Adelshausen (1855)</option>
<option value='600' title='Bad Schwalbach, LK: Rheingau-Taunus'>Adolfseck (600)</option>
<option value='1650' title='Diemelsee, LK: Waldeck-Frankenberg'>Adorf (1650)</option>
<option value='3100' title='Brensbach, LK: Odenwaldkreis'>Affhöllerbach (3100)</option>
<option value='1651' title='Edertal, LK: Waldeck-Frankenberg'>Affoldern (1651)</option>
<option value='2970' title='Wald-Michelbach, LK: Bergstraße'>Affolterbach (2970)</option>
<option value='2840' title='Weilburg, LK: Limburg-Weilburg'>Ahausen (2840)</option>
<option value='869' title='Bad Soden-Salmünster, LK: Main-Kinzig'>Ahl (869)</option>
<option value='2841' title='Limburg a. d. Lahn, LK: Limburg-Weilburg'>Ahlbach (2841)</option>
<option value='870' title='Schlüchtern, LK: Main-Kinzig'>Ahlersbach (870)</option>
<option value='1170' title='Hohenahr, LK: Lahn-Dill'>Ahrdt (1170)</option>
<option value='2111' title='Bad Sooden-Allendorf, LK: Werra-Meißner'>Ahrenberg (2111)</option>
<option value='3101' title='Beerfelden, LK: Odenwaldkreis'>Airlenbach (3101)</option>
<option value='1171' title='Fernwald, LK: Gießen'>Albach (1171)</option>
<option value='2112' title='Meißner, LK: Werra-Meißner'>Alberode (2112)</option>
<option value='2971' title='Rimbach, LK: Bergstraße'>Albersbach (2971)</option>
<option value='1652' title='Bad Wildungen, LK: Waldeck-Frankenberg'>Albertshausen (1652)</option>
<option value='1172' title='Solms, LK: Lahn-Dill'>Albshausen (1172)</option>
<option value='2441' title='Rauschenberg, LK: Marburg-Biedenkopf'>Albshausen (2441)</option>
<option value='1856' title='Guxhagen, LK: Schwalm-Eder'>Albshausen (1856)</option>
<option value='2113' title='Witzenhausen, LK: Werra-Meißner'>Albshausen (2113)</option>
<option value='2114' title='Eschwege, LK: Werra-Meißner'>Albungen (2114)</option>
<option value='601' title='Heidenrod, LK: Rheingau-Taunus'>Algenroth (601)</option>
<option value='2260' title='Kirchheim, LK: Hersfeld-Rotenburg'>Allendorf (2260)</option>
<option value='2842' title='Merenberg, LK: Limburg-Weilburg'>Allendorf (2842)</option>
<option value='1653' title='Allendorf (Eder), LK: Waldeck-Frankenberg'>Allendorf (Eder) (1653)</option>
<option value='1175' title='Gießen, LK: Gießen'>Allendorf a. d. Lahn (1175)</option>
<option value='1176' title='Allendorf (Lumda), LK: Gießen'>Allendorf a. d. Lumda (1176)</option>
<option value='2442' title='Dautphetal, LK: Marburg-Biedenkopf'>Allendorf am Hohenfels (2442)</option>
<option value='1654' title='Frankenau, LK: Waldeck-Frankenberg'>Allendorf b. Frankenau (1654)</option>
<option value='1857' title='Frielendorf, LK: Schwalm-Eder'>Allendorf/Fri. (1857)</option>
<option value='1173' title='Greifenstein, LK: Lahn-Dill'>Allendorf/G. (1173)</option>
<option value='1174' title='Haiger, LK: Lahn-Dill'>Allendorf/H. (1174)</option>
<option value='1858' title='Schwalmstadt, LK: Schwalm-Eder'>Allendorf/Sch. (1858)</option>
<option value='1655' title='Korbach, LK: Waldeck-Frankenberg'>Alleringhausen (1655)</option>
<option value='1177' title='Rabenau, LK: Gießen'>Allertshausen (1177)</option>
<option value='1050' title='Modautal, LK: Darmstadt-Dieburg'>Allertshofen (1050)</option>
<option value='11' title='Gernsheim, LK: Groß-Gerau'>Allmendfeld (11)</option>
<option value='2635' title='Lauterbach (Hessen), LK: Vogelsberg'>Allmenrod (2635)</option>
<option value='2261' title='Bad Hersfeld, LK: Hersfeld-Rotenburg'>Allmershausen (2261)</option>
<option value='52' title='Hofbieber, LK: Fulda'>Allmus (52)</option>
<option value='1859' title='Homberg (Efze), LK: Schwalm-Eder'>Allmuthshausen (1859)</option>
<option value='2443' title='Weimar (Lahn), LK: Marburg-Biedenkopf'>Allna (2443)</option>
<option value='53' title='Petersberg, LK: Fulda'>Almendorf (53)</option>
<option value='1656' title='Waldeck, LK: Waldeck-Frankenberg'>Alraft (1656)</option>
<option value='1051' title='Alsbach-Hähnlein, LK: Darmstadt-Dieburg'>Alsbach (1051)</option>
<option value='871' title='Bad Soden-Salmünster, LK: Main-Kinzig'>Alsberg (871)</option>
<option value='2636' title='Alsfeld, LK: Vogelsberg'>Alsfeld (2636)</option>
<option value='1659' title='Bad Wildungen, LK: Waldeck-Frankenberg'>Alt Wildungen (1659)</option>
<option value='2115' title='Herleshausen, LK: Werra-Meißner'>Altefeld (2115)</option>
<option value='1460' title='Baunatal, LK: Kassel'>Altenbauna (1460)</option>
<option value='1178' title='Solms, LK: Lahn-Dill'>Altenberg (1178)</option>
<option value='1860' title='Felsberg, LK: Schwalm-Eder'>Altenbrunslar (1860)</option>
<option value='1861' title='Felsberg, LK: Schwalm-Eder'>Altenburg (1861)</option>
<option value='2637' title='Alsfeld, LK: Vogelsberg'>Altenburg (2637)</option>
<option value='2116' title='Wanfried, LK: Werra-Meißner'>Altenburschla (2116)</option>
<option value='1179' title='Buseck, LK: Gießen'>Alten-Buseck (1179)</option>
<option value='1461' title='Naumburg, LK: Kassel'>Altendorf (1461)</option>
<option value='54' title='Gersfeld (Rhön), LK: Fulda'>Altenfeld (54)</option>
<option value='872' title='Sinntal, LK: Main-Kinzig'>Altengronau (872)</option>
<option value='1180' title='Laubach, LK: Gießen'>Altenhain (1180)</option>
<option value='525' title='Bad Soden am Taunus, LK: Main-Taunus'>Altenhain (525)</option>
<option value='1657' title='Haina (Kloster), LK: Waldeck-Frankenberg'>Altenhaina (1657)</option>
<option value='873' title='Linsengericht, LK: Main-Kinzig'>Altenhaßlau (873)</option>
<option value='1462' title='Wolfhagen, LK: Kassel'>Altenhasungen (1462)</option>
<option value='55' title='Ebersburg, LK: Fulda'>Altenhof (55)</option>
<option value='1181' title='Braunfels, LK: Lahn-Dill'>Altenkirchen/B. (1181)</option>
<option value='1182' title='Hohenahr, LK: Lahn-Dill'>Altenkirchen/H. (1182)</option>
<option value='1658' title='Frankenau, LK: Waldeck-Frankenberg'>Altenlotheim (1658)</option>
<option value='874' title='Freigericht, LK: Main-Kinzig'>Altenmittlau (874)</option>
<option value='1463' title='Baunatal, LK: Kassel'>Altenritte (1463)</option>
<option value='2638' title='Herbstein, LK: Vogelsberg'>Altenschlirf (2638)</option>
<option value='300' title='Altenstadt, LK: Wetterau'>Altenstadt (300)</option>
<option value='1464' title='Naumburg, LK: Kassel'>Altenstädt (1464)</option>
<option value='2444' title='Lohra, LK: Marburg-Biedenkopf'>Altenvers (2444)</option>
<option value='1052' title='Münster, LK: Darmstadt-Dieburg'>Altheim (1052)</option>
<option value='1862' title='Morschen, LK: Schwalm-Eder'>Altmorschen (1862)</option>
<option value='780' title='Weilrod, LK: Hochtaunus'>Altweilnau (780)</option>
<option value='1183' title='Herborn, LK: Lahn-Dill'>Amdorf (1183)</option>
<option value='1660' title='Diemelstadt, LK: Waldeck-Frankenberg'>Ammenhausen (1660)</option>
<option value='2445' title='Wetter (Hessen), LK: Marburg-Biedenkopf'>Amönau (2445)</option>
<option value='2446' title='Amöneburg, LK: Marburg-Biedenkopf'>Amöneburg (2446)</option>
<option value='2639' title='Alsfeld, LK: Vogelsberg'>Angenrod (2639)</option>
<option value='2640' title='Wartenberg, LK: Vogelsberg'>Angersbach (2640)</option>
<option value='3102' title='Höchst i. Odw., LK: Odenwaldkreis'>Annelsbach (3102)</option>
<option value='1184' title='Fernwald, LK: Gießen'>Annerod (1184)</option>
<option value='1661' title='Edertal, LK: Waldeck-Frankenberg'>Anraff (1661)</option>
<option value='781' title='Neu-Anspach, LK: Hochtaunus'>Anspach (781)</option>
<option value='2447' title='Kirchhain, LK: Marburg-Biedenkopf'>Anzefahr (2447)</option>
<option value='1863' title='Knüllwald, LK: Schwalm-Eder'>Appenfeld (1863)</option>
<option value='1864' title='Gilserberg, LK: Schwalm-Eder'>Appenhain (1864)</option>
<option value='2641' title='Homberg (Ohm), LK: Vogelsberg'>Appenrod (2641)</option>
<option value='1185' title='Greifenstein, LK: Lahn-Dill'>Arborn (1185)</option>
<option value='2117' title='Herleshausen, LK: Werra-Meißner'>Archfeld (2117)</option>
<option value='1465' title='Oberweser, LK: Kassel'>Arenborn (1465)</option>
<option value='2843' title='Runkel, LK: Limburg-Weilburg'>Arfurt (2843)</option>
<option value='2448' title='Weimar (Lahn), LK: Marburg-Biedenkopf'>Argenstein (2448)</option>
<option value='1053' title='Darmstadt, LK: Kreisfreie Stadt Darmstadt'>Arheilgen (1053)</option>
<option value='56' title='Dipperz, LK: Fulda'>Armenhof (56)</option>
<option value='1662' title='Bad Wildungen, LK: Waldeck-Frankenberg'>Armsfeld (1662)</option>
<option value='782' title='Schmitten, LK: Hochtaunus'>Arnoldshain (782)</option>
<option value='1865' title='Borken (Hessen), LK: Schwalm-Eder'>Arnsbach (1865)</option>
<option value='1186' title='Lich, LK: Gießen'>Arnsburg (1186)</option>
<option value='2642' title='Kirtorf, LK: Vogelsberg'>Arnshain (2642)</option>
<option value='57' title='Eiterfeld, LK: Fulda'>Arzell (57)</option>
<option value='1054' title='Modautal, LK: Darmstadt-Dieburg'>Asbach (1054)</option>
<option value='2262' title='Bad Hersfeld, LK: Hersfeld-Rotenburg'>Asbach (2262)</option>
<option value='2972' title='Wald-Michelbach, LK: Bergstraße'>Aschbach (2972)</option>
<option value='1866' title='Schwalmstadt, LK: Schwalm-Eder'>Ascherode (1866)</option>
<option value='1664' title='Vöhl, LK: Waldeck-Frankenberg'>Asel (1664)</option>
<option value='2263' title='Bebra, LK: Hersfeld-Rotenburg'>Asmushausen (2263)</option>
<option value='301' title='Niddatal, LK: Wetterau'>Assenheim (301)</option>
<option value='1187' title='Aßlar, LK: Lahn-Dill'>Aßlar (1187)</option>
<option value='602' title='Rüdesheim am Rhein, LK: Rheingau-Taunus'>Assmannshausen (602)</option>
<option value='1867' title='Neukirchen, LK: Schwalm-Eder'>Asterode (1867)</option>
<option value='12' title='Trebur, LK: Groß-Gerau'>Astheim (12)</option>
<option value='1188' title='Lahnau, LK: Lahn-Dill'>Atzbach (1188)</option>
<option value='2264' title='Rotenburg a. d. Fulda, LK: Hersfeld-Rotenburg'>Atzelrode (2264)</option>
<option value='2643' title='Mücke, LK: Vogelsberg'>Atzenhain (2643)</option>
<option value='2265' title='Neuenstein, LK: Hersfeld-Rotenburg'>Aua (2265)</option>
<option value='2119' title='Wanfried, LK: Werra-Meißner'>Aue (2119)</option>
<option value='2973' title='Bensheim, LK: Bergstraße'>Auerbach (2973)</option>
<option value='876' title='Wächtersbach, LK: Main-Kinzig'>Aufenau (876)</option>
<option value='302' title='Büdingen, LK: Wetterau'>Aulendiebach (302)</option>
<option value='2844' title='Weilmünster, LK: Limburg-Weilburg'>Aulenhausen (2844)</option>
<option value='603' title='Rüdesheim am Rhein, LK: Rheingau-Taunus'>Aulhausen (603)</option>
<option value='2845' title='Villmar, LK: Limburg-Weilburg'>Aumenau (2845)</option>
<option value='570' title='Wiesbaden, LK: Landeshauptstadt Wiesbaden'>Auringen (570)</option>
<option value='2266' title='Hohenroda, LK: Hersfeld-Rotenburg'>Ausbach (2266)</option>
<option value='1055' title='Babenhausen, LK: Darmstadt-Dieburg'>Babenhausen (1055)</option>
<option value='1663' title='Bad Arolsen, LK: Waldeck-Frankenberg'>Bad Arolsen (1663)</option>
<option value='2483' title='Bad Endbach, LK: Marburg-Biedenkopf'>Bad Endbach (2483)</option>
<option value='2314' title='Bad Hersfeld, LK: Hersfeld-Rotenburg'>Bad Hersfeld (2314)</option>
<option value='803' title='Bad Homburg v.d.Höhe, LK: Hochtaunus'>Bad Homburg v. d. Höhe (803)</option>
<option value='3151' title='Bad König, LK: Odenwaldkreis'>Bad König (3151)</option>
<option value='383' title='Bad Nauheim, LK: Wetterau'>Bad Nauheim (383)</option>
<option value='423' title='Nidda, LK: Wetterau'>Bad Salzhausen (423)</option>
<option value='222' title='Bad Salzschlirf, LK: Fulda'>Bad Salzschlirf (222)</option>
<option value='692' title='Bad Schwalbach, LK: Rheingau-Taunus'>Bad Schwalbach (692)</option>
<option value='1003' title='Bad Soden-Salmünster, LK: Main-Kinzig'>Bad Soden (1003)</option>
<option value='554' title='Bad Soden am Taunus, LK: Main-Taunus'>Bad Soden (554)</option>
<option value='2218' title='Bad Sooden-Allendorf, LK: Werra-Meißner'>Bad Sooden-Allendorf (2218)</option>
<option value='439' title='Bad Vilbel, LK: Wetterau'>Bad Vilbel (439)</option>
<option value='1837' title='Bad Wildungen, LK: Waldeck-Frankenberg'>Bad Wildungen (1837)</option>
<option value='1466' title='Bad Emstal, LK: Kassel'>Balhorn (1466)</option>
<option value='1056' title='Seeheim-Jugenheim, LK: Darmstadt-Dieburg'>Balkhausen (1056)</option>
<option value='1189' title='Mittenaar, LK: Lahn-Dill'>Ballersbach (1189)</option>
<option value='2644' title='Grebenhain, LK: Vogelsberg'>Bannerod (2644)</option>
<option value='2846' title='Merenberg, LK: Limburg-Weilburg'>Barig-Selbenhausen (2846)</option>
<option value='604' title='Schlangenbad, LK: Rheingau-Taunus'>Bärstadt (604)</option>
<option value='1665' title='Vöhl, LK: Waldeck-Frankenberg'>Basdorf (1665)</option>
<option value='58' title='Hilders, LK: Fulda'>Batten (58)</option>
<option value='1666' title='Battenberg (Eder), LK: Waldeck-Frankenberg'>Battenberg (1666)</option>
<option value='1667' title='Allendorf (Eder), LK: Waldeck-Frankenberg'>Battenfeld (1667)</option>
<option value='1668' title='Haina (Kloster), LK: Waldeck-Frankenberg'>Battenhausen (1668)</option>
<option value='2449' title='Marburg, LK: Marburg-Biedenkopf'>Bauerbach (2449)</option>
<option value='303' title='Friedberg (Hessen), LK: Wetterau'>Bauernheim (303)</option>
<option value='2410' title='Nentershausen, LK: Hersfeld-Rotenburg'>Bauhaus (2410)</option>
<option value='2267' title='Alheim, LK: Hersfeld-Rotenburg'>Baumbach (2267)</option>
<option value='13' title='Rüsselsheim, LK: Groß-Gerau'>Bauschheim (13)</option>
<option value='1467' title='Hofgeismar, LK: Kassel'>Beberbeck (1467)</option>
<option value='2268' title='Bebra, LK: Hersfeld-Rotenburg'>Bebra (2268)</option>
<option value='1190' title='Aßlar, LK: Lahn-Dill'>Bechlingen (1190)</option>
<option value='605' title='Hünstetten, LK: Rheingau-Taunus'>Bechtheim (605)</option>
<option value='2974' title='Lautertal (Odenwald), LK: Bergstraße'>Beedenkirchen (2974)</option>
<option value='2269' title='Ludwigsau, LK: Hersfeld-Rotenburg'>Beenhausen (2269)</option>
<option value='3103' title='Beerfelden, LK: Odenwaldkreis'>Beerfelden (3103)</option>
<option value='304' title='Reichelsheim (Wetterau), LK: Wetterau'>Beienheim (304)</option>
<option value='2270' title='Bad Hersfeld, LK: Hersfeld-Rotenburg'>Beiershausen (2270)</option>
<option value='1191' title='Greifenstein, LK: Lahn-Dill'>Beilstein (1191)</option>
<option value='1868' title='Malsfeld, LK: Schwalm-Eder'>Beiseförth (1868)</option>
<option value='1192' title='Mittenaar, LK: Lahn-Dill'>Bellersdorf (1192)</option>
<option value='1193' title='Hungen, LK: Gießen'>Bellersheim (1193)</option>
<option value='877' title='Steinau an der Straße, LK: Main-Kinzig'>Bellings (877)</option>
<option value='305' title='Ranstadt, LK: Wetterau'>Bellmuth (305)</option>
<option value='2450' title='Fronhausen, LK: Marburg-Biedenkopf'>Bellnhausen/Fro. (2450)</option>
<option value='2451' title='Gladenbach, LK: Marburg-Biedenkopf'>Bellnhausen/Gla. (2451)</option>
<option value='1194' title='Grünberg, LK: Gießen'>Beltershain (1194)</option>
<option value='2452' title='Ebsdorfergrund, LK: Marburg-Biedenkopf'>Beltershausen (2452)</option>
<option value='2271' title='Heringen (Werra), LK: Hersfeld-Rotenburg'>Bengendorf (2271)</option>
<option value='1669' title='Diemelsee, LK: Waldeck-Frankenberg'>Benkhausen (1669)</option>
<option value='2975' title='Bensheim, LK: Bergstraße'>Bensheim (2975)</option>
<option value='2645' title='Alsfeld, LK: Vogelsberg'>Berfa (2645)</option>
<option value='1869' title='Homberg (Efze), LK: Schwalm-Eder'>Berge (1869)</option>
<option value='2120' title='Neu-Eichenberg, LK: Werra-Meißner'>Berge (2120)</option>
<option value='1195' title='Dietzhölztal, LK: Lahn-Dill'>Bergebersbach (1195)</option>
<option value='486' title='Frankfurt am Main, LK: Kreisfreie Stadt Frankfurt am Main'>Bergen-Enkheim (486)</option>
<option value='1670' title='Bad Wildungen, LK: Waldeck-Frankenberg'>Bergfreiheit (1670)</option>
<option value='1196' title='Aßlar, LK: Lahn-Dill'>Berghausen (1196)</option>
<option value='1870' title='Spangenberg, LK: Schwalm-Eder'>Bergheim (1870)</option>
<option value='1671' title='Edertal, LK: Waldeck-Frankenberg'>Bergheim (1671)</option>
<option value='306' title='Ortenberg, LK: Wetterau'>Bergheim (306)</option>
<option value='1672' title='Battenberg (Eder), LK: Waldeck-Frankenberg'>Berghofen (1672)</option>
<option value='1468' title='Fuldabrück, LK: Kassel'>Bergshausen (1468)</option>
<option value='14' title='Groß-Gerau, LK: Groß-Gerau'>Berkach (14)</option>
<option value='487' title='Frankfurt am Main, LK: Kreisfreie Stadt Frankfurt am Main'>Berkersheim (487)</option>
<option value='2121' title='Witzenhausen, LK: Werra-Meißner'>Berlepsch-Ellerode (2121)</option>
<option value='2847' title='Weilburg, LK: Limburg-Weilburg'>Bermbach (2847)</option>
<option value='606' title='Waldems, LK: Rheingau-Taunus'>Bermbach (606)</option>
<option value='1197' title='Aßlar, LK: Lahn-Dill'>Bermoll (1197)</option>
<option value='2646' title='Grebenhain, LK: Vogelsberg'>Bermuthshain (2646)</option>
<option value='878' title='Freigericht, LK: Main-Kinzig'>Bernbach (878)</option>
<option value='1673' title='Twistetal, LK: Waldeck-Frankenberg'>Berndorf (1673)</option>
<option value='1871' title='Knüllwald, LK: Schwalm-Eder'>Berndshausen (1871)</option>
<option value='2122' title='Sontra, LK: Werra-Meißner'>Berneburg (2122)</option>
<option value='59' title='Fulda, LK: Fulda'>Bernhards (59)</option>
<option value='2647' title='Antrifttal, LK: Vogelsberg'>Bernsburg (2647)</option>
<option value='2453' title='Cölbe, LK: Marburg-Biedenkopf'>Bernsdorf (2453)</option>
<option value='2648' title='Mücke, LK: Vogelsberg'>Bernsfeld (2648)</option>
<option value='2649' title='Schlitz, LK: Vogelsberg'>Bernshausen (2649)</option>
<option value='1198' title='Reiskirchen, LK: Gießen'>Bersrod (1198)</option>
<option value='307' title='Wölfersheim, LK: Wetterau'>Berstadt (307)</option>
<option value='60' title='Fulda, LK: Fulda'>Besges (60)</option>
<option value='1872' title='Edermünde, LK: Schwalm-Eder'>Besse (1872)</option>
<option value='1199' title='Lich, LK: Gießen'>Bettenhausen (1199)</option>
<option value='1469' title='Kassel, LK: Kreisfreie Stadt Kassel'>Bettenhausen (1469)</option>
<option value='61' title='Eiterfeld, LK: Fulda'>Betzenrod (61)</option>
<option value='2650' title='Schotten, LK: Vogelsberg'>Betzenrod (2650)</option>
<option value='2454' title='Kirchhain, LK: Marburg-Biedenkopf'>Betziesdorf (2454)</option>
<option value='1873' title='Bad Zwesten, LK: Schwalm-Eder'>Betzigerode (1873)</option>
<option value='607' title='Hünstetten, LK: Rheingau-Taunus'>Beuerbach (607)</option>
<option value='1200' title='Buseck, LK: Gießen'>Beuern (1200)</option>
<option value='1874' title='Felsberg, LK: Schwalm-Eder'>Beuern (1874)</option>
<option value='2976' title='Biblis, LK: Bergstraße'>Biblis (2976)</option>
<option value='1201' title='Mittenaar, LK: Lahn-Dill'>Bicken (1201)</option>
<option value='1057' title='Bickenbach, LK: Darmstadt-Dieburg'>Bickenbach (1057)</option>
<option value='2651' title='Grebenau, LK: Vogelsberg'>Bieben (2651)</option>
<option value='725' title='Offenbach am Main, LK: Kreisfreie Stadt Offenbach am Main'>Bieber (725)</option>
<option value='879' title='Biebergemünd, LK: Main-Kinzig'>Bieber (879)</option>
<option value='15' title='Biebesheim am Rhein, LK: Groß-Gerau'>Biebesheim (15)</option>
<option value='1674' title='Hatzfeld (Eder), LK: Waldeck-Frankenberg'>Biebighausen (1674)</option>
<option value='571' title='Wiesbaden, LK: Landeshauptstadt Wiesbaden'>Biebrich (571)</option>
<option value='2272' title='Ludwigsau, LK: Hersfeld-Rotenburg'>Biedebach (2272)</option>
<option value='2455' title='Biedenkopf, LK: Marburg-Biedenkopf'>Biedenkopf (2455)</option>
<option value='572' title='Wiesbaden, LK: Landeshauptstadt Wiesbaden'>Bierstadt (572)</option>
<option value='2652' title='Alsfeld, LK: Vogelsberg'>Billertshausen (2652)</option>
<option value='1058' title='Fischbachtal, LK: Darmstadt-Dieburg'>Billings (1058)</option>
<option value='308' title='Kefenrod, LK: Wetterau'>Bindsachsen (308)</option>
<option value='309' title='Echzell, LK: Wetterau'>Bingenheim (309)</option>
<option value='1875' title='Morschen, LK: Schwalm-Eder'>Binsförth (1875)</option>
<option value='2977' title='Birkenau, LK: Bergstraße'>Birkenau (2977)</option>
<option value='1675' title='Burgwald, LK: Waldeck-Frankenberg'>Birkenbringhausen (1675)</option>
<option value='3104' title='Brombachtal, LK: Odenwaldkreis'>Birkert (3104)</option>
<option value='1202' title='Lich, LK: Gießen'>Birklar (1202)</option>
<option value='880' title='Birstein, LK: Main-Kinzig'>Birstein (880)</option>
<option value='1876' title='Neuental, LK: Schwalm-Eder'>Bischhausen (1876)</option>
<option value='2123' title='Waldkappel, LK: Werra-Meißner'>Bischhausen (2123)</option>
<option value='1203' title='Bischoffen, LK: Lahn-Dill'>Bischoffen (1203)</option>
<option value='1877' title='Spangenberg, LK: Schwalm-Eder'>Bischofferode (1877)</option>
<option value='16' title='Bischofsheim, LK: Groß-Gerau'>Bischofsheim (16)</option>
<option value='881' title='Maintal, LK: Main-Kinzig'>Bischofsheim (881)</option>
<option value='1204' title='Leun, LK: Lahn-Dill'>Biskirchen (1204)</option>
<option value='1205' title='Leun, LK: Lahn-Dill'>Bissenberg (1205)</option>
<option value='310' title='Echzell, LK: Wetterau'>Bisses (310)</option>
<option value='62' title='Hosenfeld, LK: Fulda'>Blankenau (62)</option>
<option value='2124' title='Sontra, LK: Werra-Meißner'>Blankenbach (2124)</option>
<option value='2273' title='Bebra, LK: Hersfeld-Rotenburg'>Blankenheim (2273)</option>
<option value='1206' title='Wetzlar, LK: Lahn-Dill'>Blasbach (1206)</option>
<option value='311' title='Ortenberg, LK: Wetterau'>Bleichenbach (311)</option>
<option value='2653' title='Homberg (Ohm), LK: Vogelsberg'>Bleidenrod (2653)</option>

Changes.txt
You can get an overview on the changes in Mapbender from the Version History:

http://www.mapbender.org/index.php/Version_History

The mapbender trac provides different reports. The reports give detailed information about the enhancements

Changelog 2.7.2
http://trac.osgeo.org/mapbender/query?status=closed&group=resolution&milestone=2.7.2+release
790	Mapbender StartUpParams are not handled	
840	IE: WFS featureType configuration - popup helptext ok button not displayed size incorrect	
765	jquery ui dialog (jumps)	
786	savewms and loadwmc in my updated guis	
807	mod_usemap  divs um_img and um_draw are missing	
816	Digitize: Wrong mouse position in scrolled page in IE	
817	New Constant in mapbender.conf	
818	Digitize: Does not work properly in scrolled IE page	
819	WMC Update to 2.7	
821	Mod_digitize_splitline don't intersect closed lines.	
827	Dynamic print frame: provide possibility to cancel a running print	
828	Dynamic print frame: delete temporary files after pdf creation	
829	Dynamic print frame: log WMS request in mb_log table in Mapbender db	
830	Dynamic print frame: avoid reload with ie	
831	Dynamic print frame: scale modification	
834	administration - remove application from category does not work	
836	admin2_de fails to load with IE8	
838	wrong entry new at the top of the treeGDE	
839	login.php application overview - change switch on mouseover to click	
841	IE: WFS featureType configuration - attribute columns - wrong number of header columns and columns	
846	layer with sld settings not displayed...	
852	scaleSelect has to refer to ../plugins/mb_selectScale.js and does not need e_attributes	
853	update script overwrites the user defined EPSG options	
854	open Legend with a button in a dialog	
858	integration of new applications PortalAdmin and PortalAdmin_DE	
859	french translation update in table translation for textes with '	
796	mismatching i18-Strings de	
808	mb_background doesn't work in 2.7	
812	Wrong Value in SQL-Update-Script	
813	Clean 'layer_load_count'	
814	URL of OL-Link to WMC	
815	Title of OL-Link from WMC	
844	guis looses Category when renamed	
848	2.7.x: Count of failure-login by users is not counted	
851	initial icon for Layer-Group wrong	
823	Show Icons in real Size	
833	Picture for showCoords not transparent (blue_red)	
782	table mb_meetingpoint does not exist	
842	add createCategory to Administzration and Administration_DE	
811	spatial search and digitize conflict	
850	FeatureInfo Dialog jumping arround	
527	HTTP authentication for OWS Proxy	
837	function getAvailableWfsForGeom(geom wfsConf) does not check for gml:MultiPolygonPropertyType and other element_types gml:* (with GeoServer 2.1.0)

-------------------------------------------------------


Changelog 2.7.1
http://trac.osgeo.org/mapbender/query?status=closed&group=resolution&milestone=2.7.1+release
778	WFS administration
791	copy application - gui_lyer_title are not copied to new application
799	gui_digitize update of a geometry does not work
801	setbackground - background wms is not displayed
806	load wfs doesn't load describeFeatureType information
506	Use WMS monitor from www.geoportal.rlp.de
661	Northarrow Print Overview
715	Cutting two lines does not work as expected when
720	login.php creates a session even if the user is not logged in
734	temporary load GeoRSS and KML
743	reloadTree triggers a single map request for every layer
746	Layertree for GeoRSS and KML files GeoRSS
747	WFS conf Layertree for selection
749	setbackground - mb_background
773	senseful description for new metadata applications
774	admin_wmc_metadata does not load when no wmc is in the database
776	slippy map (Browser overflow)
777	change application description does not work
784	WMS Metadata fails to start - authentication issue?
787	Monitoring runs on database errors
788	some scripts refer to x_geoportal instead of geoportal
789	create and edit user - filter for users does not filter correctly
792	featureInfoResults in dialog windows are displayed at the same position
793	Can't insert Basepoint
794	Digitize/update feature shows all configured Featuretypes
800	resizeMapsize does not work on start
802	slider in template_print is not displayed correctly
699	wms status in treGDE
739	Installation from script
756	enhanced Metadatainformations for service and layer - check Abo alert a notice
780	slippy map in Internet Explorer ...
781	MousewheelZoom and selArea
795	undefined targets of toogelModule after upgrade
775	translation mistake in AdWMSFromFilteredListAjax
503	Digitize module pop-ups
810	WFS update has problem with updating featuretypes
316	Mapbender treeGDE Observations
451	select tool problem
454	polish translation for Mapbender
459	Administration Interface - TranslationBuilder
485	Update Governance pages
476	scale parameter for getlegendgraphic dropped by owsproxy
785	setBBOX doesn't work
500	Bug in WFS-Update
779	Digitizing
803	Administration_DE and Administration: Delete WFS Conf module does not load

-------------------------------------------------------


Changelog 2.7
http://trac.osgeo.org/mapbender/report/48
20	addWMS beim Login
82	Problems with dynamicOverview and setBackground
344	zebra: handle gui-categories in guilist / adminitstrate gui categories
365	calculate new Scalehint if epsg 4326 is choosed
389	Configurable Treefolder
412	Copyright transfer to OSGeo
418	Digitizing: add one-click clone button
424	Update points instantly
426	Allow objects and arrays in element vars
437	Open save dialogue immediately after a geometry has been created
438	AddWMSfromFilteredList_Ajax - added new script to handle the button
439	treeGDE translations i18n
447	digitizing: Nontransactional geometries are not display
450	cannot create polygon, gml interpretation error
478	Layers auto-checked in treeGDE initially
481	Add WFS conf profiles
489	Zoom in on double click
497	arrange Element does not work with some browser
498	gazetteerWFS send search request on load of the application
499	gazetteerWFS - search without param selection

http/extensions/mbCatalogueSearchClient/src/conf/parameters.php
<?php

$configuration = array(
    'title'       => 'Geoportal Suche',
    'placeholder' => 'Bitte Suchbegriff eingeben...',
    'search'      => array(
        'geoportal' => array(
            'autocomplete' => array(
                'url' => 'http://www.geoportal.rlp.de/mapbender/geoportal/mod_getCatalogueKeywordSuggestion.php?',
                'maxResults' => 15
            ),

            'searchUrl' => 'http://www.geoportal.rlp.de/mapbender/php/mod_callMetadata.php?',
            'downloadUrl' => 'http://geoportal.rlp.de/mapbender/php/mod_getDownloadOptions.php?id=',
            'showMapUrl' => 'http://geoportal.rlp.de/portal/karten.html?',
            'title'     => 'Interaktive Daten',
            'class'     => 'src/search/geoportal/Search',
            'form' => array(
                'map' => array(
                    'center'  => array( //only EPSG:4326 is supported
                        'lat' => 49.9,
                        'lon' => 7.3
                    ),
                    'zoom' => 7,
                    'wms'  => array(
                        'url'    => 'http://osm-demo.wheregroup.com/service',
                        'layers' => array('osm'),
                        'format' => 'image/png',
                    )
                )
            ),
            'resources' => array(
                'dataset' => 'Datensätze',
                'wms'     => 'Darstellungsdienste',
                'wfs'     => 'Such- und Download- und Erfassungsmodule',
                'wmc'     => 'Kartenzusammenstellungen'
            )
        )
    )
);
